examples/tutorials/data/hawc.py

```diff
@@ -0,0 +1,268 @@
+# Load the tables
+
+data_path = "$GAMMAPY_DATA/hawc/crab_events_pass4/"
+hdu_filename = f"hdu-index-table-{energy_estimator}-Crab.fits.gz"
+obs_filename = f"obs-index-table-{energy_estimator}-Crab.fits.gz"
+
+# there is only one observation table
+obs_table = ObservationTable.read(data_path + obs_filename)
+
+# we read the hdu index table of fHit bin number 6
+fHit = 6
+hdu_table = HDUIndexTable.read(data_path + hdu_filename, hdu=fHit)
+
+
+######################################################################
+# From the tables, we can create a Datastore
+
+data_store = DataStore(hdu_table=hdu_table, obs_table=obs_table)
+
+data_store.info()
+
+
+######################################################################
+# There is only one observation
+
+obs = data_store.get_observations()[0]
+
+######################################################################
+# Select and peek events
+
+obs.events.peek()
+
+
+######################################################################
+# Peek the energy dispersion
+
+obs.edisp.peek()
+
+######################################################################
+# Peek the psf
+obs.psf.peek()
+
+######################################################################
+# Peek the background for one transit
+plt.figure()
+obs.bkg.reduce_over_axes().plot(add_cbar=True)
+
+######################################################################
+# Peek the effective exposure for one transit
+plt.figure()
+obs.aeff.reduce_over_axes().plot(add_cbar=True)
+
+
+######################################################################
+# Data reduction into a MapDataset
+# --------------------------------
+#
+# We will now produce a MapDataset using the data from one of the
+# fHit bins. In order to use all bins, one just needs to repeat this
+# process inside of a for loop that modifies the variable fHit.
+
+
+######################################################################
+# First we define the geometry and axes
+
+# create the energy reco axis
+# Note that this axis is the one used to create the background model map. If
+# different edges are used, the MapDatasetMaker will interpolate between
+# them, which might lead to unexpected behavior.
+energy_axis = MapAxis.from_edges(
+    [1.00, 1.78, 3.16, 5.62, 10.0, 17.8, 31.6, 56.2, 100, 177, 316] * u.TeV,
+    name="energy",
+    interp="log",
+)
+
+
+# and energy true axis
+energy_axis_true = MapAxis.from_energy_bounds(
+    1e-3, 1e4, nbin=140, unit="TeV", name="energy_true"
+)
+
+
+# create a geometry around the Crab location
+geom = WcsGeom.create(
+    skydir=SkyCoord(ra=83.63, dec=22.01, unit="deg", frame="icrs"),
+    width=6 * u.deg,
+    axes=[energy_axis],
+    binsz=0.05,
+)
+
+
+######################################################################
+# Define the makers we will use
+
+maker = MapDatasetMaker(selection=["counts", "background", "exposure", "edisp", "psf"])
+safemask_maker = SafeMaskMaker(methods=["aeff-max"], aeff_percent=10)
```

Comment:
No need for `bkg-peak` here ?

Reply:
no by default the background template scales well in each energy bin.

Reply:
During fitting, do you  keep the background norm/tilt free?

Reply:
no, the background in HAWC is usually very well characterized and so there is in principle no need for this correction (if the background is estimated ok)